packages/sdk/src/addresses/index.ts

```diff
@@ -119,25 +119,31 @@ export async function getAddresses({
 export function getAccountDataFromHdNode({
   hdNode,
   format = "legacy",
-  network = "testnet"
+  network = "testnet",
+  accountIndex = 0,
+  index = 0
```

Comment:
nit: rename to maintain consistency.

```suggestion
  addressIndex = 0
```